src/Shared/ExceptionHandling.cs

```diff
@@ -351,6 +354,31 @@ internal static void DumpExceptionToFile(Exception ex)
                 }
             }
         }
+
+        /// <summary>
+        /// Returns the content of any exception dump files modified
+        /// since the provided time, otherwise returns an empty string.
+        /// </summary>
+        internal static string ReadAnyExceptionFromFile(DateTime fromTimeUtc)
+        {
+            var builder = new StringBuilder();
```

Comment:
Nit: You could use the ReusableStringBuilder here. I wouldn't care too much since this abnormal termination. Though VS does have a metric which counts some form of memory at VS exit (though it probably does not include node crashes)

Reply:
After this point, VS presumably won't work any more anyway.. at least I'm assuming the BuildManager can't be reused, who knows what state everything is in.